source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -324,6 +342,10 @@ export class Carousel extends Container {
         this._containerForAdorners = containerForAdorners;
 
         cardLevelContainer.appendChild(containerForAdorners);
+        
+        const navigationContainer: HTMLElement = document.createElement("div");
```

Comment:
I don't think this will be a breaking change. I didn't do anything with this div except for styling within our own container. Should I check with Somya to be sure?

Reply:
that would be great! if the look changed at Widget Board; they will be mad :-)